nautobot/extras/tests/test_customfields.py

```diff
@@ -509,7 +478,6 @@ def test_get_single_object_without_custom_field_data(self):
         self.assertEqual(response.data["name"], self.locations[0].name)
         # A model directly instantiated via the ORM does NOT automatically receive custom field default values.
         # This is arguably a bug.
```

Comment:
Can add a link to https://github.com/nautobot/nautobot/issues/3312 here specifically :)